src/pages/CartPage.tsx

```diff
@@ -36,17 +35,12 @@ const CartPage = () => {
       )}
       <main
         className={classNames(
-          'container mx-auto flex flex-col px-4 py-6 tablet:px-6 desktop:w-[1200px] desktop:px-8',
+          'container mx-auto flex flex-col p-4 pb-6 tablet:px-6 desktop:w-[1200px]',
           { 'blur pointer-events-none': showModal },
         )}
       >
-        <span
-          onClick={() => navigate(-1)}
-          className="select-none cursor-pointer flex items-center gap-1"
-        >
-          <FiChevronLeft />
-          Back
-        </span>
+        <BreadCrumb />
+
```

Comment:
we don't need it on cart page as I saw on figma

Reply:
For next reviewers: we agreed to leave breadcrumps